app/email.go

```diff
@@ -26,8 +26,9 @@ func (a *App) SendChangeUsernameEmail(oldUsername, newUsername, email, locale, s
 	bodyPage := a.NewEmailTemplate("email_change_body", locale)
 	bodyPage.Props["SiteURL"] = siteURL
 	bodyPage.Props["Title"] = T("api.templates.username_change_body.title")
-	bodyPage.Html["Info"] = utils.TranslateAsHtml(T, "api.templates.username_change_body.info",
+	bodyPage.Props["Info"] = T("api.templates.username_change_body.info",
 		map[string]interface{}{"TeamDisplayName": a.Config().TeamSettings.SiteName, "NewUsername": newUsername})
+	bodyPage.Props["Warning"] = T("api.templates.email_warning")
```

Comment:
`api.templates.email_warning` is not defined in the translation keys (I think)

Reply:
Yes, you are right. it's missing in `i18n/en.json`.
`api.templates.email_warning` should be `If you did not make this change, please contact the system administrator.`. I'll add it.

Thank you.